### ISSUE TITLE
Image scaling and normalization

### ISSUE THREAD

**Issue by enrico310786** [SHI-Labs/Compact-Transformers#61]
Hi,

if i want to use the model cct_14_7x2_224 pretrained on the Imagenet dataset, do i have to right normalize the images using the mean and standard dev of imagenet dataset, respectively (0.485, 0.456, 0.406) (0.229, 0.224, 0.225). Rigth? Before this normalization, do i have to make the rescaling of the pixel values from [0:255] to [0:1] ?

Thanks

**Comment by alihassanijr** [comment]
Hello,

Technically, yes. Not just our model, but rather any statistical model by definition aims to generalize to samples drawn from the same distribution as the one it was trained on, and normalization is a key part of that. If one were to use a pre-trained model for inference on unseen data, it is typically best to use the same normalization as the training data. However, that is not necessary if you transfer the model (re-train or fine-tune) to new data.

As for the scaling, torchvision's ToTensor transform converts PIL images (what they typically start out as because of the loaders) in [0,255] pixel scale to tensors in the [0,1] scale, so just having the basic transforms (ToTensor and Normalize) should really be all you need for inference.
